src/backend/aspen/workflows/nextstrain_run/export_test.py

```diff
@@ -68,12 +102,24 @@ def cli(
             "num_sequences": num_sequences,
             "num_included_samples": num_included_samples,
         }
-        group = session.query(Group).first()
+        if group_name:
+            group = session.query(Group).filter(Group.name == group_name).first()
+        else:
+            group = session.query(Group).first()
+        if not group:
```

Comment:
you're not using sqlalchemy 2.0 style querying?

Reply:
also would one() make more sense than first()?

Reply:
`one()` will freak out if the query *could* return multiple rows. And you're right that I should convert the query style.